src/main/java/christmas/constant/PromotionCalendar.java

```diff
@@ -0,0 +1,39 @@
+package christmas.constant;
+
+import java.util.Calendar;
+
+public enum PromotionCalendar {
+    PERIOD_MONTH(2023, Calendar.DECEMBER, 1);
+
+    private final Calendar calendar;
+
+    PromotionCalendar(int year, int month, int day) {
+        Calendar calendar = Calendar.getInstance();
+        calendar.set(year, month, day, 0, 0, 0);
+        this.calendar = calendar;
+    }
```

Comment:
[개인적인 의견입니다!]

파라미터가 이렇게 많은 상황에서
적절한 개행은, 코드 가독성을 높여준다고 생각해요!

```suggestion
    PromotionCalendar(
        int year, 
        int month, 
        int day
) {
        Calendar calendar = Calendar.getInstance();
        calendar.set(year, month, day, 0, 0, 0);
        this.calendar = calendar;
    }
```